File: 1486-xor-operation-in-an-array/1486-xor-operation-in-an-array.c
int xorOperation(int n, int start){
    int r = 0;
    int *nums;
    nums = (int*)malloc(sizeof(int) * n);
    for(int i = 0; i < n; ++i)
    {
        nums[i] = start + 2*i;
        r = r ^ nums[i];
    }
    return r;
}

File: 1486-xor-operation-in-an-array/1486-xor-operation-in-an-array.swift
class Solution {
    func xorOperation(_ n: Int, _ start: Int) -> Int {
        var xorResult = start   //start + 2*0
        
        for i in 1..<n {
            var currentValue = start + 2*i
            xorResult ^= currentValue
        }
        return xorResult
    }
}